### ISSUE TITLE
`/` only show file completion

### ISSUE THREAD

**Issue by sucrecacao** [alexherbo2/snippets.kak#12]
After opening `/tmp/test.rb, running `snippets-enable`, `require-module ruby` , I type `/` in insert mode and the only result I get is the file completion. 

Any hint on how I can debug this ?

**Comment by alexherbo2** [comment]
Have you built the snippets?

``` sh
snippets build
```

**Comment by alexherbo2** [comment]
@sucrecacao Try to play with the command-line interface.

See the [snippets#examples] section.

[snippets#examples]: https://github.com/alexherbo2/snippets#examples

**Comment by sucrecacao** [comment]
`snippets` works perfectly on the command line. 

Which hook or command `/` is suppose to trigger ?

**Comment by sucrecacao** [comment]
Ok I see. I have to type at least 2 char to see the snippets in the completion, otherwise I only see file completion. 
Also, I havn't try all ruby snippets but from my quick test I can see that:
`class` `begin` appear in the completion menu
`if` `do` `loop` does apear in the completion menu.

Otherwise, when I type `/<snippet-name>` and `<a-ret>` it works as expected and print the whole snippet.

So to summarize, just a few snippet appear in the menu and they appear after typing `/` + at least 2 characters

**Comment by alexherbo2** [comment]
Fixed

**Comment by sucrecacao** [comment]
The completion work for ruby and other already defined language, but not for my new python snippets

**Comment by alexherbo2** [comment]
Have you rebuilt the snippets?

**Comment by sucrecacao** [comment]
Yes, actually even the ruby one's are not in the autocomplet. I just tested with "class" but "class" is autocompleted as a language keywork. "protected-def" didn't worked

**Comment by sucrecacao** [comment]
I though lsp could overwrite the completion but the completion doesn't work neither when lsp is disabled

**Comment by alexherbo2** [comment]
Try with the command-line tool for debugging.

**Comment by sucrecacao** [comment]
Command line tool work fine
```
> snippets get snippet python def
{"scope":"python","name":"def","content":"def {{function_name}}({{args}}):\n\t{{function_body}}","path":"/home/joe/.config/snippets/base/python/def"}
```

Snippets works well too. Is just the auto-complet menu that doesn't show the candidats.

What could I test with the cli?

**Comment by alexherbo2** [comment]
I dunno 😭

Maybe kcr dependency.

**Comment by sucrecacao** [comment]
I'm trying to update kcr, which made me update crystal which gave me a bug: https://github.com/alexherbo2/kakoune.cr/issues/13

**Comment by sucrecacao** [comment]
All your tools are great but every time I try to update one I have to do a lot of work and update all the dependency chain which is always a bit complicated
